routing/dht/dht_net.go

```diff
@@ -227,3 +228,23 @@ func (ms *messageSender) SendRequest(ctx context.Context, pmes *pb.Message) (*pb
 
 	return mes, nil
 }
+
+func (ms *messageSender) ctxReadMsg(ctx context.Context, mes *pb.Message) error {
+	t := time.NewTimer(time.Second * 30)
```

Comment:
I think we should collect all of those timeouts (and create a one that would be standard), this way if we need to make IPFS work over higher latency for some reason it will be much easier.